src/Raven.Client/Documents/Operations/Replication/PullReplicationDefinition.cs

```diff
@@ -99,9 +99,9 @@ internal void Validate(bool useSsl)
             }
         }
 
-        public ExternalReplication ToExternalReplication(ReplicationInitialRequest request, long taskId)
+        public PullReplicationAsHub ToPullReplicationAsHub(ReplicationInitialRequest request, long taskId)
```

Comment:
can we change this? this is public Client Api.

Reply:
removed